packages/a/ajv-draft-04.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "ajv-draft-04",
+  "description": "Another JSON Schema Validator specifically for Draft-04 support",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ajv-validator/ajv-draft-04.git"
+  },
+  "keywords": [
+    "JSON",
+    "schema",
+    "validator",
+    "jsonschema",
+    "json-schema",
+    "draft-04"
+  ],
+  "license": "MIT",
+  "filename": "index.js",
+  "autoupdate": {
+    "source": "npm",
+    "target": "ajv-draft-04",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*"
```

```suggestion
          "*.@(js?(.map)|d.ts)"
```
We generally try to avoid having open glob-stars and wildcards, as it can lead to files be included that were not meant to be on cdnjs. It looks like we've just got some JS & typings files here.